documentation/modules/operators/ref-operator-cluster.adoc

```diff
@@ -217,6 +217,50 @@ Enables or disables features and functionality controlled by xref:ref-operator-c
 `STRIMZI_POD_SECURITY_PROVIDER_CLASS`:: Optional.
 Configures the pluggable `PodSecurityProvider` class which can be used to provide the security context configuration for Pods and containers.
 
+[[STRIMZI_LEADER_ELECTION_ENABLED]]
+`STRIMZI_LEADER_ELECTION_ENABLED`:: Optional, default `false`.
+Enables or disables the leader election which allows to run additional operator instances as a hot-stand-by.
+Disabled by default.
+
+`STRIMZI_LEADER_ELECTION_LEASE_NAME`:: Required when leader election is enabled.
+The name of the Kubernetes Lease resource which is used for the leader election.
+
+`STRIMZI_LEADER_ELECTION_LEASE_NAMESPACE`:: Required when leader election is enabled.
+The namespace where the Kubernetes Lease resource used for the leader election will be created.
+You can use the link:https://kubernetes.io/docs/tasks/inject-data-application/downward-api-volume-expose-pod-information/#the-downward-api[Kubernetes Downward API^] to configure it to the namespace where the operator is deployed.
++
+[source,yaml,options="nowrap"]
+----
+env:
+  - name: STRIMZI_LEADER_ELECTION_LEASE_NAMESPACE
+    valueFrom:
+      fieldRef:
+        fieldPath: metadata.namespace
+----
+
+`STRIMZI_LEADER_ELECTION_IDENTITY`:: Required when leader election is enabled.
+Configures the identity of given operator instance used during the leader election.
+Should be unique for each operator instance
+You can use the link:https://kubernetes.io/docs/tasks/inject-data-application/downward-api-volume-expose-pod-information/#the-downward-api[Kubernetes Downward API^] to configure it to the name of the operator pod.
++
+[source,yaml,options="nowrap"]
+----
+env:
+  - name: STRIMZI_LEADER_ELECTION_IDENTITY
+    valueFrom:
+      fieldRef:
+        fieldPath: metadata.name
+----
+
+`STRIMZI_LEADER_ELECTION_LEASE_DURATION_MS`:: Optional, default `15` seconds.
```

Comment:
I see are describing the defaults for other `_MS` env var in ms. For example ...

> STRIMZI_FULL_RECONCILIATION_INTERVAL_MS
Optional, default is 120000 ms. The interval between periodic reconciliations, in milliseconds.

I would do the same here as 15000 ms and for the others.

Reply:
Makes sense. I fixed it for all 3 `_MS` options I added.